Amazon/Marketplace/v7153/templates/jfrog-artifactory-ec2-instance.template.yaml

```diff
@@ -92,7 +86,7 @@ Mappings:
     us-west-1:
       "Artifactory7153": ami-0afdf2d63fc2c0895
     us-west-2:
-      "Artifactory7153": ami-09274621c30cae078
+      "Artifactory7153": ami-03c0e58806c907dee
```

Comment:
We definitely need to update us-east-1 image. it needs to be in that region to be scanned. Also they take that AMI to replicate.